docs/learning/howto/apache2-proxy-gssapi.md

```diff
@@ -0,0 +1,104 @@
+# Apache 2.4 Reverse Proxy + custom URL + GSSAPI
+
+This section summarizes one possible configuration setup that can be used to expose Rundeck 
+from an apache2 reverse proxy with a custom url path and GSSAPI authentication (with delegated headers).
+In this way Rundeck can be hosted together with other applications in the same url structure.
+
+The final goal is to expose to the outside world Rundeck as `http://srv.example.com/rundeck/`
+while the actual server (in this example) is running in the same VM but bound locally as `http://127.0.0.1:4440/rundeck/`.
+
+In addition authentication will be transparently handled by apache2 via GSSAPI, 
+which is an easy way to implement SSO where Active Directory (or Kerberos) is available.
+
+
+
+# Prerequisites
+
+We assume that inside the VM:
+* Rundeck has been already set up with all the necessary configuration options for DB, logging, etc.
+* apache2 is running with the necessary modules enabled (proxy, gssapi, etc)
+* a valid keytab has been generated and contains the correct principal keys (in this case HTTP/srv.example.com@EXAMPLE.COM)
+
+Please refer the relevant documentation for more details:
+*   [apache2 proxy balancer](https://httpd.apache.org/docs/2.4/mod/mod_proxy_balancer.html)
+*   [apache2 gssapi](https://github.com/gssapi/mod_auth_gssapi)
+
+It is important to test the correct behavior of the GSSAPI configuration before attempting to apply it to Rundeck.
+
+
+
+# Configuration Steps
+
+The configuration on apache2 for exposing Rundeck can be set up in different ways, in this case we will  use the proxy balancer module.
+The configuration is straightforward, however particular care must be applied to absence of an ending `/` in the various urls.
+
+
+The `GSSAPI` (and related `RequestHeader`) configuration is of course optional 
+and if applied should be customized with the actual location of the keytab files and domain name `EXAMPLE.COM`.
+In addition the stripping of the domain name `EXAMPLE.COM` from the `Uiid` field should be updated as well, if needed.
+
+Last but not least, the look up of user roles, necessary for the correct execution of the authorization process on Rundeck side, 
+is implemented by reading one file named as the user principal and located under the folder `/etc/apache2/rundeck-roles/`.
+The file contains one row with the list of roles specific to the user, for example `admin,user`.
+
+This approach is very simple and perhaps suitable only for a small user base 
```

Comment:
This comment should likely go up at the top in the description of the article/solution.